src/main/java/com/synopsys/integration/alert/channel/jira/JiraIssueHandler.java

```diff
@@ -71,8 +71,6 @@
 
 public class JiraIssueHandler {
     public static final String DESCRIPTION_CONTINUED_TEXT = "(description continued...)";
-    public static final String TODO_STATUS_CATEGORY_KEY = "new";
-    public static final String DONE_STATUS_CATEGORY_KEY = "done";
```

Comment:
I snuck this in because these constants were unused and duplicated in another file.